File: src/B_2476/B_2476.java
package B_2476;

import java.io.*;
import java.util.*;

public class B_2476 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());
        int res = 0 ;
        StringTokenizer st;
        while(N-->0){
            st = new StringTokenizer(br.readLine()," ");
            int[] arr = new int[3];
            for(int i=0; i<3; ++i){
                arr[i] = Integer.parseInt(st.nextToken());
            }
            res = Math.max(res, money(arr[0],arr[1],arr[2]));

        }
        System.out.print(res);

    }
    static int money(int a,int b,int c){
        if(a == b && b == c){
            return 10_000 + a * 1_000;
        }
        else if(a == b){
            return 1_000 + a * 100;
        }
        else if(a == c){
            return 1_000 + a * 100;
        }
        else if(c == b){
            return 1_000 + b * 100;
        }
        else{
            return Math.max(Math.max(a,b), Math.max(b,c)) * 100;
        }
    }

}

File: src/B_2476/B_2476.kt
package B_2476

import java.io.*
import java.util.*
fun main(args: Array<String>) {
    val br = BufferedReader(InputStreamReader(System.`in`))
    var N = br.readLine().toInt()
    var res = 0
    var st: StringTokenizer
    while (N-- > 0) {
        st = StringTokenizer(br.readLine(), " ")
        val arr = IntArray(3)
        for (i in 0..2) {
            arr[i] = st.nextToken().toInt()
        }
        res = res.coerceAtLeast(money(arr[0], arr[1], arr[2]))
    }
    print(res)
}

fun money(a: Int, b: Int, c: Int): Int {
    return if (a == b && b == c) {
        10000 + a * 1000
    } else if (a == b) {
        1000 + a * 100
    } else if (a == c) {
        1000 + a * 100
    } else if (c == b) {
        1000 + b * 100
    } else {
        Math.max(Math.max(a, b), Math.max(b, c)) * 100
    }
}

